requirements.txt

```diff
@@ -1 +1 @@
--r requirements/prod.txt
+--r requirements/prod.txt
```

Comment:
It should be `-r` instead of `--r`

Reply:
Can you tell me any reason for this?

Reply:
@vaibhavsingh97 It is as it is , I haven't made changes in it.